src/views/ManageList.jsx

```diff
@@ -1,7 +1,63 @@
-export function ManageList() {
+import { useState } from 'react';
+import { addItem } from '../api';
+
+export function ManageList({ listPath }) {
+	const initialFormState = {
```

Comment:
I like how you made the form state an object vs. separate pieces of state for each input!

Reply:
praise: Agreed! It feels clean to have it all wrapped up in one.

Reply:
I feel like it could come in handy if we want to add more details / properties to our shopping list items as well!